internal/data/assets_test.go

```diff
@@ -735,6 +742,7 @@ func Test_GetAssetsPerReceiverWallet(t *testing.T) {
 					ID:          receiverY.ID,
 					Email:       receiverY.Email,
 					PhoneNumber: receiverY.PhoneNumber,
+					ExternalID: receiverY.ExternalID,
```

Comment:
I think your IDE is not indenting this correctly. Maybe this is the old spaces vs tab indentation issue.